### PR TITLE
OCLOMRS-955 and OCLOMRS-956: Add missing permissions checks for organizations

### PR DESCRIPTION
**Pull request by suruchee**
Missing Permissions Check for Organization Member Management:OCLOMRS-955 and Missing Permissions Check for "Edit Organization" button:OCLOMRS-956"

### CODE REVIEW

File: src/apps/organisations/pages/ViewOrgPage.tsx [hunk 1]
```diff
@@ -29,8 +29,12 @@ import {Grid, makeStyles, createStyles} from "@material-ui/core";
 import {EditButton} from "../../containers/components/EditButton";
 import {getPrettyError} from "../../../utils";
 import { AppState } from "../../../redux";
+import {APIOrg, APIProfile, canModifyContainer, profileSelector} from "../../authentication";
```
Comment (hadijahkyampeire) [inline src/apps/organisations/pages/ViewOrgPage.tsx:32]: Please remove this import of `canModifyContainer` since you are not using it.

**Review by hadijahkyampeire — COMMENTED** [review]
Thanks @suruchee , this works well for me. I just have minor suggestions to maintain a good code base, otherwise, I will refer to @ibacher to confirm if this is what is required.

**Comment by suruchee** [comment]
Thanks, @hadijahkyampeire  for the review, I have updated the PR with improved formatting.

**Review by hadijahkyampeire — APPROVED** [review]
LGTM, well done @suruchee

**Review by ibacher — APPROVED** [review]

**Pull request merged**